### PR TITLE
Remove KA references from errata tests

### PR DESCRIPTION
**Pull request by vsedmik**
This PR just removes katello-agent references from errata tests.

### CODE REVIEW

**Comment by vsedmik** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_errata.py -k test_positive_install_in_hc

**Review by damoore044 — APPROVED** [review]

File: tests/foreman/api/test_errata.py [hunk 1]
```diff
@@ -75,51 +75,13 @@ def custom_repo(module_org, module_lce, module_cv, activation_key):
...
-                    host.hostname,
-                )
-            )
+def _install_package(clients, package_name):
+    """Install package via SSH"""
+    for client in clients:
+        result = client.run(f'yum install -y {package_name}')
+        assert result.status == 0
+        result = client.run(f'rpm -q {package_name}')
+        assert result.status == 0
```
Comment (adarshdubey-star) [inline tests/foreman/api/test_errata.py:84]: Non blocking :
```suggestion
        assert client.run(f'yum install -y {package_name}').status == 0
        assert client.run(f'rpm -q {package_name}').status == 0
```

**Review by adarshdubey-star — APPROVED** [review]
Ack, with some non-blocking suggestions

**Comment by vsedmik** [comment]
Addressed @adarshdubey-star 's comment.

**Comment by adarshdubey-star** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_errata.py -k test_positive_install_in_hc

**Review by synkd — APPROVED** [review]
ACK pending PRT results

File: tests/foreman/api/test_errata.py [hunk 2]
```diff
@@ -75,51 +75,11 @@ def custom_repo(module_org, module_lce, module_cv, activation_key):
...
-                    package_name,
-                    'installed on' if expected_installed else 'removed from',
-                    host.hostname,
-                )
-            )
+def _install_package(clients, package_name):
+    """Install package via SSH"""
+    for client in clients:
+        assert client.run(f'yum install -y {package_name}').status == 0
+        assert client.run(f'rpm -q {package_name}').status == 0
```
Comment (Gauravtalreja1) [inline tests/foreman/api/test_errata.py:82]: why having  a separate func just to install the package, instead we could just assert the yum/dnf command on client in the test itself, wdyt?
```suggestion
assert rhel_contenthost.run(f'yum install -y {package_name}').status == 0
```

Comment (vsedmik) [reply tests/foreman/api/test_errata.py:82]: heck yeah, after the simplification the function does not make much sense

**Comment by vsedmik** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_errata.py -k test_positive_install_in_hc

File: tests/foreman/api/test_errata.py (hunk 2 above)
Comment (Gauravtalreja1) [reply tests/foreman/api/test_errata.py]: thank you!

**Review by Gauravtalreja1 — APPROVED** [review]

**Pull request merged**